meta-mender-demo/recipes-connectivity/qemu-connectivity-helper/files/qemu-connectivity-helper.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Prolong the shutdown, so that Qemu SLiRP TCP connections have time to be brought down.
+After=network-online.target
+Before=mender-client.service mender-gateway.service
+
+[Service]
+ExecStart=/bin/true
+RemainAfterExit=yes
+ExecStop=/bin/sleep 10
```

Comment:
It's probably premature to optimize this, but if this approach works, it would be great if we could experiment with getting this number down to 2 or something. There are a lot of reboots during a test run...

Reply:
So, do you still think we should start out at 10?

Reply:
My suggestion: Keep 10 for now, and get it submitted, but try to lower it in a followup PR afterwards.